src/Agent.Worker/Release/ContainerFetchEngine/FetchEngine.cs

```diff
@@ -240,6 +240,11 @@ private void LogStatistics()
         {
             try
             {
+                lock (_lock)
```

Comment:
do we need Lock? ExecutionContext.write() will take lock on write to log file on disk, web console line queue is a concurrent queue.